www/install/php/Update-22.04.0-beta.1.php

```diff
@@ -164,90 +162,95 @@
-        $customConfiguration = [
-            "trusted_client_addresses" => !empty($result['openid_connect_trusted_clients'])
-                ? explode(',', $result['openid_connect_trusted_clients'])
-                : [],
-            "blacklist_client_addresses" => !empty($result['openid_connect_blacklist_clients'])
-                ? explode(',', $result['openid_connect_blacklist_clients'])
-                : [],
-            "base_url" => !empty($result['openid_connect_base_url']) ? $result['openid_connect_base_url'] : null,
-            "authorization_endpoint" => !empty($result['openid_connect_authorization_endpoint'])
-                ? $result['openid_connect_authorization_endpoint']
-                : null,
-            "token_endpoint" => !empty($result['openid_connect_token_endpoint'])
-                ? $result['openid_connect_token_endpoint']
-                : null,
-            "introspection_token_endpoint" => !empty($result['openid_connect_introspection_endpoint'])
-                ? $result['openid_connect_introspection_endpoint']
-                : null,
-            "userinfo_endpoint" => !empty($result['openid_connect_userinfo_endpoint'])
-                ? $result['openid_connect_userinfo_endpoint']
-                : null,
-            "endsession_endpoint" => !empty($result['openid_connect_end_session_endpoint'])
-                ? $result['openid_connect_end_session_endpoint']
-                : null,
-            "connection_scopes" => explode(" ", $result['openid_connect_scope']),
-            "login_claim" => !empty($result['openid_connect_login_claim'])
-                ? $result['openid_connect_login_claim']
-                : null,
-            "client_id" => !empty($result['openid_connect_client_id'])
-                ? $result['openid_connect_client_id']
-                : null,
-            "client_secret" => !empty($result['openid_connect_client_secret'])
-                ? $result['openid_connect_client_secret']
-                : null,
-            "authentication_type" => $result['openid_connect_client_basic_auth'] === '1'
-                ? 'client_secret_basic'
-                : 'client_secret_post',
-            "verify_peer" => $result['openid_connect_verify_peer'] === '1' ? false : true // '1' is Verify Peer disable
-        ];
-
-        $statement2 = $pearDB->prepare(
-            "INSERT INTO provider_configuration (`type`,`name`,`custom_configuration`,`is_active`,`is_forced`)
-            VALUES ('openid','openid', :customConfiguration, :isActive, :isForced)"
-        );
-        $statement2->bindValue(':customConfiguration', json_encode($customConfiguration), \PDO::PARAM_STR);
-        $statement2->bindValue(':isActive', $isActive ? '1' : '0', \PDO::PARAM_STR);
-        $statement2->bindValue(':isForced', $isForced ? '1' : '0', \PDO::PARAM_STR);
-        $statement2->execute();
-
+    $result = $statement->fetchAll(\PDO::FETCH_ASSOC);
+    if (!empty($result)) {
```

Comment:
not needed. In PHP 8, fetchAll returns always an array
```suggestion
```

Reply:
as discussed with you, i'll let it as the DELETE query to remove the old configuration should be executed only in the case where we have found something